### PR TITLE
added component state in details.js - Task #13

### PR DESCRIPTION
**Pull request by maxgdev**
Please review and accept pull request. Thanks

### CODE REVIEW

File: src/Details/Details.js [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+import React, {useState, useEffect} from 'react';
 import { Link } from 'react-router-dom';
 import './Details.css';
+
 export default function Details() {
+    let [message, setMessage] = useState(
+      'Quisque diam odio, cursus et ante a, varius commodo libero. Fusce eget metus urna. Vestibulum vulputate finibus tortor, sed vestibulum quam volutpat eget. Praesent consequat sapien ac porta fermentum. Proin vestibulum augue sit amet auctor scelerisque. Etiam euismod dignissim mauris ac commodo'
+    );
+    useEffect(()=> {
+      setTimeout(()=> setMessage('Coming Soon :)'), 3000);
```
Comment (GrahamHewett) [inline src/Details/Details.js]: All seems correct Max, just watch the elegance on things like arrow functions. () => is the correct spacing not ()=>

File: src/Details/Details.js [hunk 2]
```diff
@@ -1,13 +1,19 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
```
Comment (GrahamHewett) [inline src/Details/Details.js]: Same comment as below. Spaces inside the curly brackets for named imports { useState, useEffect }

File: src/Details/Details.js (hunk 1 above)
Comment (maxgdev) [reply src/Details/Details.js]: Point taken

File: src/Details/Details.js (hunk 2 above)
Comment (maxgdev) [reply src/Details/Details.js]: Indeed.

**Pull request merged**